### ISSUE TITLE
minor suggestions about images

### ISSUE THREAD

**Issue by azimut** [edannenberg/gentoo-bb#100]
Some ideas I got working with containers and might be consider here. If not valid here feel free to close this.

* about nginx-php, might be this image can be de-coupled, and have one image for php-fpm and other for nginx. The official docker images for wordpress took this approach. Which also follows the pattern of having only one binary running per container. Additionally share the same fs could be a vector of attack (if permissions are too permissive) https://blog.tarq.io/root-your-box-with-w3tc-and-nginx/ Although this brings some extra volume linking needed between then.
* about postgres, the entrypoint script is posix so in theory it doesn't need bash

**Comment by jbergstroem** [comment]
@azimut the idea is that you create your own containers -- see the bundled ones as examples!

**Comment by edannenberg** [comment]
@jbergstroem Agreed, but on the other hand the example images should be best practice, so general improvements should always be welcome.

@azimut I'll fix the postgres script but I don't really have the time for the nginx-php refactor at the moment, will happily accept a PR though.
